cmd/clusterctl/client/client_test.go

```diff
@@ -132,10 +132,10 @@ func newFakeClient(configClient config.Client) *fakeClient {
 	}
 
 	var clusterClientFactory = func(i ClusterClientFactoryInput) (cluster.Client, error) {
-		// converting the client.Kubeconfig to cluster.Kubeconfig alias
-		k := cluster.Kubeconfig(i.kubeconfig)
+		// converting the client.Kubeconfig tproviderr.Kubeconfig alias
```

Comment:
nit: seems like there was replace error here.
```suggestion
		// converting the client.Kubeconfig to cluster.Kubeconfig alias
```